doc/nrf/drivers/eth_rtt.rst

```diff
@@ -0,0 +1,44 @@
+.. _lib_eth_rtt:
+
+Ethernet over RTT driver
+########################
+
+This driver transfers Ethernet frames using the SEGGER J-Link debugger Real-Time Transfer feature.
+
+It allows to sniff network data and to stream log information from a running application to an RTT client.
+
+It is meant to be used for debugging and testing purposes only.
+It requires a software application on the PC side, developed specifically to handle the frames transferred via RTT channels.
```

Comment:
Maybe it is better to write:
```suggestion
It requires a software application on the PC side, developed specifically to handle the data stream transferred via RTT channels.
```
to make it clear the data stream has to be decoded and validated to get the Ethernet frames out of it.

Reply:
fixed